database/migrations/2017_10_24_085111_create_order_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateOrdersTable extends Migration
```

Comment:
Did you change the name of the class after your initial migration? When I am migrating it asks for `CreateOrderTable` and fails